src/coreclr/vm/prestub.cpp

```diff
@@ -2357,7 +2357,7 @@ PCODE MethodDesc::DoPrestub(MethodTable *pDispatchingMT, CallerGCMode callerGCMo
     pCode = DoBackpatch(pMT, pDispatchingMT, FALSE);
 
 Return:
-#if defined(FEATURE_INTERPRETER) && defined(FEATURE_JIT)
+#if defined(FEATURE_INTERPRETER) && !defined(TARGET_WASM)
```

Comment:
The #endif below needs its comment updated
Also, why this change? It's not immediately evident to me. I'm guessing it's that wasm doesn't have the stubs we need? A TODO/FIXME comment might be good so we know to come back and change this later, or if it's going to be like this forever a brief comment explaining why could be nice.

Reply:
The initial condition was set to omit stub generation for WASM, since it was the only platform running only with the interpreter and without the JIT. This PR statically links the interpreter and excludes the JIT on iOS, so I updated the condition to apply on iOS and continue omitting WASM until support is implemented.

Reply:
I will add a comment explaining this.